### PR TITLE
Add Dutch Translation

### PR DESCRIPTION
**Pull request by lodu**
For some words, for instance repository, there is no good Dutch alternative. However there is a lot of so called "Anglicism" aka borrowing words from English. So all of my colleagues will use the word repository when speaking Dutch.

### CODE REVIEW

File: ModernFlyouts/MultilingualResources/ModernFlyouts.nl.xlf
Comment (lodu) [inline ModernFlyouts/MultilingualResources/ModernFlyouts.nl.xlf:144]: this can either be 'aanvullend' or 'aanvullende' depending on the sentence/usage.

**Comment by ShankarBUS** [comment]
Thank you so much @lodu!

I'll let my mate @Samuel12321 review and merge this PR.

File: ModernFlyouts/MultilingualResources/ModernFlyouts.nl.xlf
Comment (lodu) [inline ModernFlyouts/MultilingualResources/ModernFlyouts.nl.xlf:397]: I have never heard this word used before, always system tray, but I think some people won't understand system tray.

**Comment by Samuel12321** [comment]
Hi thanks for all your help @lodu,  have added this to the 0.9 release. Will merge shortly  once text is finalised.

**Review by Samuel12321 — APPROVED** [review]
Will merge for 0.9

**Comment by Samuel12321** [comment]
@lodu problem with this PR caused build to fail. This needs to be fixed before merging again.

**Pull request merged**
